routes/index.js

```diff
@@ -1,8 +1,14 @@
 import { Router } from "express";
 import userRouters from "./users";
+import articleRoutes from "./articles";
+import checkAuth from "../middlewares/checkAuth";
+import commentRoutes from "./comments";
 
 const router = Router();
 
+// PUBLIC ROUTES
 router.use(userRouters);
+// PRIVATE(PROTECTED) ROUTES
+router.use(checkAuth, articleRoutes, commentRoutes);
```

Comment:
Not all endpoints related to the article requires authentication, for example: fetching one article and fetch all articles `checkAuth` should be on specific routes, not all endpoints related to the articles. @Musigwa

Reply:
Okay, I'll fix that